backend/fleet_management/management/commands/populate_database.py

```diff
@@ -1,80 +1,122 @@
 
-        self.stdout.write(self.style.SUCCESS("Creating 5 projects"))
+        projects = [
+            ProjectFactory.make(
+                title="Default Project", description="Default Project", country="UA",
+            )
+        ]
+        info("Creating 5 projects")
         for _ in tqdm(range(5)):
-            ProjectFactory.create()
-
-        self.stdout.write(self.style.SUCCESS("Creating 50 drives"))
-        all_drivers = list(User.objects.filter(groups=driver_group))
-        all_passengers = list(User.objects.filter(groups=passenger_group))
-        all_projects = list(Project.objects.all())
-
+            projects.append(ProjectFactory.make())
+
+        DriveFactory.make(
+            driver=drivers[0],
+            project=projects[0],
+            passenger=passengers[0],
+            car=cars[0],
+            start_mileage=1,
+            end_mileage=100,
+            description="Default verified drive",
+            start_location="Start",
+            end_location="End",
+            is_verified=True,
+        )
+        DriveFactory.make(
+            driver=drivers[0],
+            project=projects[0],
+            passenger=passengers[0],
+            car=cars[0],
+            start_mileage=1,
+            end_mileage=100,
+            description="Default unverified drive",
+            start_location="Start",
+            end_location="End",
+            is_verified=False,
+        )
+        info("Creating 50 drives")
         for _ in tqdm(range(50)):
-            DriveFactory.create(
-                passenger=random.choice(all_passengers),
-                project=random.choice(all_projects),
-                driver=random.choice(all_drivers),
+            DriveFactory.make(
+                passenger=random.choice(passengers),
+                project=random.choice(projects),
+                driver=random.choice(drivers),
             )
 
-        self.stdout.write(self.style.SUCCESS("Database successfully populated"))
+        info("Database successfully populated")
+        info("=" * 60)
 
-        self.stdout.write("=" * 50)
-        self.stdout.write(self.style.SUCCESS("Newly created drivers:"))
-        for index, username in enumerate(drivers):
-            self.stdout.write(
-                self.style.SUCCESS(
-                    "{index}. {username}".format(index=index, username=username)
-                )
-            )
-
-        self.stdout.write(self.style.SUCCESS("Newly created passengers:"))
-        for index, username in enumerate(passengers):
-            self.stdout.write(
-                self.style.SUCCESS(
-                    "{index}. {username}".format(index=index, username=username)
-                )
-            )
+        info(f"{'Newly created drivers (username : password)':>56}")
+        for index, user in enumerate(drivers, start=1):
+            info(f"{index:02d}. {user.username:>40} : {UserFactory.password}")
```

Comment:
Yup, it can on rare occasions.I think its better to let it wrap around than to pad too much to right (it can wrap there as well if terminal window is small enough).